pkg/activator/handler/handler_test.go

```diff
@@ -29,6 +29,16 @@ import (
 	. "github.com/knative/pkg/logging/testing"
 	"github.com/knative/serving/pkg/activator"
 	"github.com/knative/serving/pkg/activator/util"
+	fakeservingclientset "github.com/knative/serving/pkg/client/clientset/versioned/fake"
+	servinginformers "github.com/knative/serving/pkg/client/informers/externalversions"
+	"github.com/knative/serving/pkg/queue"
+	"sync"
```

Comment:
sync should be grouped with stdlib imports.
I saw this elsewhere as well (mix&match imports).

Reply:
Done